src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -252,6 +252,20 @@ protected function prepareForDenormalization($data)
     }
 
     /**
+     * Returns the method to use to construct an object. This method must be either
+     * the object constructor or static.
+     *
+     * @param string           $class           a class name
+     * @param \ReflectionClass $reflectionClass
+     *
+     * @return \ReflectionMethod|null
+     */
+    protected function getConstructor($class, \ReflectionClass $reflectionClass)
```

Comment:
I think that makes sense in case you want to support different factory methods depending on the input.